riscv/riscv.mk.in

```diff
@@ -14,6 +14,7 @@ riscv_hdrs = \
 	devices.h \
 	dts.h \
 	mmu.h \
+    base_cfg.h \
```

Comment:
Indentation doesn't match surrounding code (check for tabs vs spaces)

Reply:
Fixed, thanks.